stixcore/tmtc/tests/test_packets.py

```diff
@@ -61,6 +62,13 @@ def test_tc_packet():
                 for key in TC_DATA_HEADER_STRUCTURE.keys() if not key.startswith('spare')])
 
 
+def test_packet_laszlo(idb):
```

Comment:
maybe a better name or description in comments?

Reply:
removed test